app/controllers/notes_controller.rb

```diff
@@ -3,13 +3,16 @@
 class NotesController < ApplicationController
   def create
     @need = Need.find(params[:need_id])
-    @note = @need.notes.create(note_params.merge(user: current_user))
+    params = note_params
+    params.merge!(body: 'No details captured') if note_params[:body].blank?
```

Comment:
This is probably fine for the time being, if you create a trello card in the backlog about sorting the rules out for this. I think ideally we'd require either a category (assuming non phone calls have an empty category; otherwise we'd require a non-"note" category) _or_ body. But it's fine for now.

Reply:
Trello card created.